entrypoint.sh

```diff
@@ -35,7 +35,14 @@ then
 
 fi
 
+# Working Directory
+workingDirectory=$4
+
+if [[ -n $workingDirectory ]]; then
```

Comment:
I know this is picky but I think you should either test for the existence of the target directory or for the success of the `cd`. Otherwise a config error could get missed / ignored.

Reply:
When we add the check for directory exists here, the script will not signal an error and continue in the root working directory. I think it should fail when the workingDirectory does not exist (as is currently the case). I could improve the error message by providing an error like e.g. `ERROR: working-directory does not exist`

Reply:
That would work. 

I didn't see the `set -e` last time around. 

I'm not a fan of `set -e` in non-trivial scripts cos it doesn't provide good diagnostics.